### DISCUSSION TITLE
是否支持Elasticsearch的存储

### DISCUSSION THREAD

**Discussion by BruceChinese** [juicedata/juicefs#2807] [General]
使用腾讯云 存储是COS    JuiceFS是否能当作Elasticsearch的存储 性能方面是否有影响

**Comment by davies** [comment]
JuiceFS+COS can be used as the warm storage for ES.

**Reply by WinstonCCC** [reply]
可以参考JuiceFS 在携程海量冷数据场景下的实践 https://mp.weixin.qq.com/s/n3rkD-0nxTCHEEYWGe_GzAttps://mp.weixin.qq.com/s/n3rkD-0nxTCHEEYWGe_GzA 和 Elasticsearch 存储成本省 60%，稿定科技干货分享 https://mp.weixin.qq.com/s/lxqaEhYHoqP7N_FpXY6L6g
